website/docs/reference/artifacts/run-results-json.md

```diff
@@ -5,7 +5,28 @@ sidebar_label: "Run results"
 
 **Current schema**: [`v4`](https://schemas.getdbt.com/dbt/run-results/v4/index.html)
 
-**Produced by:** [`build`](/reference/commands/build) [`compile`](/reference/commands/compile) [`docs generate`](/reference/commands/cmd-docs) [`seed`](/reference/commands/seed) [`snapshot`](/reference/commands/snapshot) [`test`](/reference/commands/test) [`run`](/reference/commands/run)
+<VersionBlock lastVersion="1.6">
```

Comment:
Hey @aranke I wonder if this should be 1.5? Otherwise both blocks might show up?

Reply:
Also, I wonder if you could version only the new command for 1.6 like this (which might make the code easier to read):
```
**Produced by:**
  [`build`](/reference/commands/build)
  [`compile`](/reference/commands/compile)
  [`docs generate`](/reference/commands/cmd-docs)
  [`run`](/reference/commands/run)
 <VersionBlock firstVersion="1.6"> [`run-operation`](/reference/commands/run-operation)</VersionBlock>
  [`seed`](/reference/commands/seed)
  [`snapshot`](/reference/commands/snapshot)
  [`test`](/reference/commands/test)
```

Reply:
going to fix this as it's looking like this and snapshots, test, seed don't appear in v1.5 
![Screenshot 2023-06-07 at 12 02 33](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/384ec3a6-4bd5-4397-b11e-e89ff3c00507)